### PR TITLE
Add CIVIC plugin

### PR DESCRIPTION
**Pull request by cmdcolin**
This was a pretty basic plugin but fleshes out the list a bit

### CODE REVIEW

**Comment by cmdcolin** [comment]
We may want to make these plugins like this one, ucsc, mygene.info, etc have an ability to add tracks to the users session or something similar via menu options or a widget

**Comment by cmdcolin** [comment]
right now if a user added this to their instance via sessionPlugins they wouldn't have much purpose

File: plugins.json [hunk 1]
```diff
@@ -62,6 +62,15 @@
       "url": "https://unpkg.com/jbrowse-plugin-ideogram/dist/jbrowse-plugin-ideogram.umd.production.min.js",
       "license": "Apache License 2.0",
       "image": "https://raw.githubusercontent.com/GMOD/jbrowse-plugin-list/main/img/ideogram-screenshot-fs8.png"
+    },
+    {
+      "name": "CIVIC",
+      "authors": ["Colin Diesh"],
+      "description": "JBrowse 2 plugin for fetching data from the CIVIC clinical interpretation of cancer variants",
+      "location": "https://github.com/cmdcolin/jbrowse-plugin-civic",
```
Comment (rbuels) [inline plugins.json]: is this URL right? can't see it. is the repo private?

File: plugins.json [hunk 2]
```diff
@@ -62,6 +62,15 @@
...
+      "url": "https://unpkg.com/jbrowse-plugin-civc/dist/jbrowse-plugin-civic.umd.production.min.js",
+      "license": "Apache License 2.0",
+      "image": "https://raw.githubusercontent.com/GMOD/jbrowse-plugin-list/main/img/civic-screenshot-fs8.png"
```
Comment (rbuels) [inline plugins.json:73]: 404 when I tried to look at this image, is it there?

File: plugins.json [hunk 3]
```diff
@@ -62,6 +62,15 @@
       "url": "https://unpkg.com/jbrowse-plugin-ideogram/dist/jbrowse-plugin-ideogram.umd.production.min.js",
       "license": "Apache License 2.0",
       "image": "https://raw.githubusercontent.com/GMOD/jbrowse-plugin-list/main/img/ideogram-screenshot-fs8.png"
+    },
+    {
+      "name": "CIVIC",
+      "authors": ["Colin Diesh"],
+      "description": "JBrowse 2 plugin for fetching data from the CIVIC clinical interpretation of cancer variants",
+      "location": "https://github.com/cmdcolin/jbrowse-plugin-civic",
+      "url": "https://unpkg.com/jbrowse-plugin-civc/dist/jbrowse-plugin-civic.umd.production.min.js",
```
Comment (rbuels) [inline plugins.json]: 404 from unpkg, has this been published?

**Review by rbuels — CHANGES_REQUESTED** [review]

File: plugins.json (hunk 3 above)
Comment (cmdcolin) [reply plugins.json]: had a typo in url, fixed now

File: plugins.json (hunk 2 above)
Comment (cmdcolin) [reply plugins.json:73]: it refers to main branch but image will work when merged

File: plugins.json (hunk 1 above)
Comment (cmdcolin) [reply plugins.json]: fixed link

**Pull request merged**